velox/exec/Operator.h

```diff
@@ -124,6 +124,12 @@ struct OperatorStats {
 
   CpuWallTiming finishTiming;
 
+  // Metric for operators to report CPU time spent on background activities
```

Comment:
nit: drop "Metric for operators to report"

`// CPU time spent ...`

Reply:
Fixed.